en/docs/references/troubleshooting/add-non-ballerina-files-to-choreo.md

```diff
@@ -0,0 +1,51 @@
+# Add Non-Ballerina files to Choreo
```

Comment:
Since this is the title we need to go with title caps:
Add Non-Ballerina files to Choreo

Also, IMO looking at the title makes it seem like this is something that we need to have in a section related to best practices so that users can particularly avoid experiencing the issue if done as suggested when using non-Ballerina files in an integration.